docs/Components/BaseComponent.md

```diff
@@ -99,13 +99,13 @@ print(base_component.external_forces)
 >>> {'force1': <function force1 at 0x0000025EBCAA8D30>}
 ```
 ## *add_external_moment_function*:
-This method is responsible for appending an external moment function to the component and it should return the magnitude of the moment only.
+This method is responsible for appending an external moment function to the component and it should return a float representing the magnitude of the moment.
```

Comment:
I think we could yet improve this statement, separating in two phrases:

"This method is responsible for appending an external moment function to the component. The appended function should return a float representing the magnitude of the moment."